Makefile

```diff
@@ -114,27 +114,19 @@ packages:
 # Generate tests
 # ==============
 
-# Generate new Dune tests from the problems in
-# the directory tests/.
-gentest: $(wildcard tests/**/*)
-	# TODO: There should be an option to gentest to avoid printing the
-	# dune.inc file and only touch the expected files in this call.
-	dune exec -- tools/gentest.exe tests >/dev/null
```

Comment:
I think we no longer have a way to generate automatically new expected files for new tests now. Do we need to add them manually?

Reply:
Nope, it still work and I checked it. After putting your new file, you have just to run `make runtest` and `gentest` will create an empty expected file for you and compare it with the result. Then you can run `make promote` if you are happy with the output ;)

Reply:
uhm actually, `make runtest` will create a new file in `_build/default/test/`, so you cannot promote it :/ Ok we have to fix that.